src/mds/MDLog.h

```diff
@@ -308,9 +309,12 @@ class MDLog {
   void write_head(MDSContext *onfinish);
 
   // -- helper functions
-  void _log_estimated_complete_time(utime_t start_time, uint64_t start_pos);
+  void _log_estimated_complete_time(ceph::coarse_mono_time start_time, uint64_t start_pos);
 
   // -- events --
   LogEvent *cur_event = nullptr;
+  ceph::coarse_mono_time start_time;
```

Comment:
Is there a need to have these are class variables? These are used in `get_replay_estimated_complete_time` and these can be passed as arguments to the function just like its done for `_log_estimated_complete_time`.

Reply:
OK. I see what has happened - start_time and start_pos are updated in _replay_thread and the time estimate is calculated when `get_replay_estimated_complete_time` gets caller (from Beacon.cc).

(It was a bit confusing with these changes been done b/w successive commits)